### PR TITLE
Add GET /about API support

### PR DESCRIPTION
**Pull request by idolize**
Hi thanks for this project!

I'm trying to use the [`about.get` API](https://developers.google.com/drive/api/reference/rest/v3/about/get) to load some information about the user, so I thought I'd go ahead and submit a PR to make that an option in the library.

I tried to follow all the existing conventions and added tests as well (although I did not add this function to the example app). I also verified all tests (including old tests) pass.

test results

### CODE REVIEW

**Comment by darrarski** [comment]
Hello @idolize! Thanks for your great contribution. There is only one small issue with unit tests: `GetAboutTests.testGetAbout` is failing because of a wrong assertion against the request URL. It's easy to fix:

**file:** `Tests/GoogleDriveClientTests/GetAboutTests.swift`
**line:** 61

```diff
- let url = URL(string: "https://www.googleapis.com/drive/v3/about")!
+ let url = URL(string: "https://www.googleapis.com/drive/v3/about?fields=\(About.apiFields)")!
```

I guess you just forgot to include the `fields` query parameter to the expected URL. With the above change, tests will pass. I'm happy to merge the PR and release a new version that includes your changes once the issue is fixed.

Thanks again for contributing!

File: Tests/GoogleDriveClientTests/GetAboutTests.swift [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+      }
+    )
+
+    let about = try await GetAbout()
+
+    await didRefreshToken.withValue {
+      XCTAssertEqual($0, 1)
+    }
+    await httpRequests.withValue {
+      let url = URL(string: "https://www.googleapis.com/drive/v3/about")!
```
Comment (darrarski) [inline Tests/GoogleDriveClientTests/GetAboutTests.swift]:
```suggestion
      let url = URL(string: "https://www.googleapis.com/drive/v3/about?fields=\(About.apiFields)")!
```

**Review by darrarski — CHANGES_REQUESTED** [review]
Nice contribution!

Please fix the expected URL in `GetAboutTests.testGetAbout` test (see my comment for details).

**Comment by idolize** [comment]
Thanks, I made a change and forgot to update that part of the test.

Should be fixed now!

image

**Comment by idolize** [comment]
Also, I wasn't sure if you want to add this function to the README as well or not

**Comment by darrarski** [comment]
Thanks! I will update the README once merged.

**Review by darrarski — APPROVED** [review]
Great job, thank you!

**Pull request merged**